cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,59 @@
-describe('', () => {
+/// <reference types='cypress' />
+
+describe('Conduit article flow', () => {
+  let user;
+  let articleData;
+  let slug;
+
   before(() => {
+    cy.visit('');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.task('generateDataForArticle').then((generateDataForArticle) => {
+      articleData = generateDataForArticle;
+    });
+
+    beforeEach(() => {
```

Comment:
The `beforeEach` hook is inside the `before` hook, which might not be what you intended. Usually, `beforeEach` is used at the same level as `it` to run before each test case.